app/package.json

```diff
@@ -57,9 +58,12 @@
     "react-switch": "^5.0.1",
     "react-transition-group": "^4.4.1",
     "react-virtualized-auto-sizer": "^1.0.2",
+    "react-wordcloud": "^1.2.7",
+    "recharts": "^1.8.5",
     "redux": "^4.0.5",
     "redux-batched-actions": "^0.5.0",
     "redux-saga": "^1.1.3",
+    "reselect": "^4.0.0",
```

Comment:
We shouldn't need to have an explicit dependency on `reselect` since redux toolkit includes/re-exports it: https://redux-toolkit.js.org/api/createSelector

Reply:
it does not have `createSelectorCreator` or `defaultMemoize` however

Reply:
Good call, nevermind